Common/src/CConfig.cpp

```diff
@@ -1777,11 +1777,11 @@ void CConfig::SetConfig_Options() {
   /* DESCRIPTION: Activate The adaptive CFL number. */
   addBoolOption("CFL_ADAPT", CFL_Adapt, false);
   /* !\brief CFL_ADAPT_PARAM
-   * DESCRIPTION: Parameters of the adaptive CFL number (factor down, factor up, CFL limit (min and max), acceptable linear residual )
+   * DESCRIPTION: Parameters of the adaptive CFL number (factor down, factor up, CFL limit (min and max)[, acceptable linear residual][, starting iteration]).
    * Factor down generally <1.0, factor up generally > 1.0 to cause the CFL to increase when the under-relaxation parameter is 1.0
    * and to decrease when the under-relaxation parameter is less than 0.1. Factor is multiplicative. \ingroup Config*/
   default_cfl_adapt[0] = 1.0; default_cfl_adapt[1] = 1.0; default_cfl_adapt[2] = 10.0; default_cfl_adapt[3] = 100.0;
-  default_cfl_adapt[4] = 0.001;
+  default_cfl_adapt[4] = 0.001; default_cfl_adapt[5] = 0.0;
```

Comment:
```suggestion
  default_cfl_adapt[4] = 0.001; default_cfl_adapt[5] = 1.0;
```

Reply:
We should start at iteration 0 by default or else all adaptive cfl cases will fail.